docs/de/docs/index.md

```diff
@@ -293,175 +287,175 @@ def update_item(item_id: int, item: Item):
     return {"item_name": item.name, "item_id": item_id}
 ```
 
-The server should reload automatically (because you added `--reload` to the `uvicorn` command above).
+Der Server sollte automatisch neu geladen werden (weil Sie oben `--reload` zum Befehl `uvicorn` hinzugefügt haben).
 
-### Interactive API docs upgrade
+### Upgrade der interaktiven API-Dokumentation
 
-Now go to <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>.
+Gehen Sie nun auf <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>.
 
-* The interactive API documentation will be automatically updated, including the new body:
+* Die interaktive API-Dokumentation wird automatisch aktualisiert, einschließlich des neuen Body:
 
 ![Swagger UI](https://fastapi.tiangolo.com/img/index/index-03-swagger-02.png)
 
-* Click on the button "Try it out", it allows you to fill the parameters and directly interact with the API:
+* Klicken Sie auf die Taste "Try it out", damit können Sie die Parameter ausfüllen und direkt mit der API interagieren:
 
 ![Swagger UI interaction](https://fastapi.tiangolo.com/img/index/index-04-swagger-03.png)
 
-* Then click on the "Execute" button, the user interface will communicate with your API, send the parameters, get the results and show them on the screen:
+* Klicken Sie dann auf die Taste "Execute", die Benutzeroberfläche kommuniziert mit Ihrer API, sendet die Parameter, holt die Ergebnisse ab und zeigt sie auf dem Bildschirm an:
 
 ![Swagger UI interaction](https://fastapi.tiangolo.com/img/index/index-05-swagger-04.png)
 
-### Alternative API docs upgrade
+### Upgrade der alternativen API-Dokumentation
 
-And now, go to <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>.
+Und nun gehen Sie auf <a href="http://127.0.0.1:8000/redoc" class="external-link" target="_blank">http://127.0.0.1:8000/redoc</a>.
 
-* The alternative documentation will also reflect the new query parameter and body:
+* Die alternative Dokumentation wird auch den neuen Abfrageparameter und -inhalt widerspiegeln:
 
 ![ReDoc](https://fastapi.tiangolo.com/img/index/index-06-redoc-02.png)
 
-### Recap
+### Zusammenfassung
 
-In summary, you declare **once** the types of parameters, body, etc. as function parameters. 
+Zusammengefasst deklarieren Sie **einmal** die Typen von Parametern, Body, etc. als Funktionsparameter.
 
-You do that with standard modern Python types.
+Das machen Sie mit modernen Standard-Python-Typen.
 
-You don't have to learn a new syntax, the methods or classes of a specific library, etc.
+Sie müssen keine neue Syntax, die Methoden oder Klassen einer bestimmten Bibliothek usw. lernen.
```

Comment:
Better remove 'die':
`Sie müssen keine neue Syntax, Methoden oder Klassen einer bestimmten Bibliothek usw. lernen.`

Reply:
Stimme zu, ist in meinem Edit nicht enthalten.